cds/orgsvc/C#/DiscoveryService/README.md

```diff
@@ -11,7 +11,7 @@ You must first set the `username` and `password` variables in the `SampleProgram
 
 ## What this sample does
 
-This sample uses the SDK Assembly `DiscoveryServiceProxy` to query the discovery service with a user's credentials to determine which environments they can connect with.
+This sample uses the SDK Assembly `CrmServiceClient` to query the global discovery service with a user's credentials to determine which environments they can connect with.
 
```

Comment:
Should be "This sample uses a `CrmServiceClient` object...". CrmServiceClient is not an assembly.

Reply:
You could also make 'CrmServiceClient' a link to https://docs.microsoft.com/dotnet/api/microsoft.xrm.tooling.connector.crmserviceclient

Reply:
Fixed